packages/remix-cloudflare-pages/package.json

```diff
@@ -21,7 +21,7 @@
     "@cloudflare/workers-types": "^3.2.0"
```

Comment:
`remix-cloudflare-pages` could probably also support v2 as well?

```suggestion
    "@cloudflare/workers-types": "^2.0.0 || ^3.0.0"
```
CC/ @GregBrimble @jacob-ebey

Reply:
As mentioned above if you are on CF Pages then you need the V3 types. V2 does not include the necessary types for pages.

Reply:
@edmundhung I've updated the suggestion, so that the peerDependency is less strict.